.github/workflows/deploy.yml

```diff
@@ -38,6 +38,11 @@ jobs:
       run: |
         sudo apt install -y dpkg-dev gpg wget
 
+    - name: Import GPG key Secret Key (neo-project)
+      run: |
+        echo -n "${{ secrets.GPG_KEY_PRIVATE }}" | \
+        gpg --import --passphrase ${{ secrets.GPG_KEY_PASSWORD }} --batch
```

Comment:
```suggestion
      run: |
         echo -n "${{ secrets.GPG_KEY_PRIVATE }}" | gpg \
         --batch --yes --pinentry-mode loopback \
         --passphrase "${{ secrets.GPG_KEY_PASSWORD }}" \
         --import
```
If you make --import first, it will ignore --passphrase and force you inputing password  with UI pop-up, like:
<img width="519" height="218" alt="image" src="https://github.com/user-attachments/assets/218aabc1-9182-468e-8926-a9e4a4ffad46" />

Add `--pinentry-mode loopback + --passphrase` to make it more safe and avoid interaction with UI pop-up as well.